dali/ft/filecopy.cpp

```diff
@@ -1224,6 +1224,8 @@ void FileSprayer::calculateSprayPartition()
                 distributedTarget->setECL(recStru.str());
         }
     }
+    if (compressedInput && compressOutput && streq(encryptKey.str(),decryptKey.str()))
```

Comment:
Theoretically yes, they can be differ from DFUPlus help:
        encrypt=<password> -- optional, encrypt target
        decrypt=<password> -- optional, decrypt source

Yes, you can copy uncompressed source file to a compressed target.